dev-python/sympy/sympy-1.11.1-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Computer Algebra System in pure Python"
+HOMEPAGE="https://www.sympy.org/"
+SRC_URI="https://github.com/sympy/sympy/archive/${P}.tar.gz -> ${P}.gh.tar.gz"
+S="${WORKDIR}/${PN}-${P}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 ~riscv x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos"
```

Comment:
You can use e.g. `ekeyword ~all sympy-1.11.1-r1.ebuild` to lower the keywords to `~arch`.

Reply:
@mgorny ah OK, I didn't realize that [revision bumps upgrades need keyword downgrades](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades).